test/controllers/recipe_controller_test.rb

```diff
@@ -0,0 +1,17 @@
+require "test_helper"
+
+describe RecipeController do
+  it "should get index" do
+    VCR.use_cassette('search_file') do
+      get recipes_path
+       must_respond_with :success
```

Comment:
There are some interesting cases you're not covering here:

- What happens if no search term is provided?
- What if the search returns no results?

It might also be worthwhile to add some tests around the paging parameters:

- What happens if they're absent?
- Do you get different results when they change?
- What if you send a bogus value, like a negative page number?